src/vcModals.cpp

```diff
@@ -54,30 +54,29 @@ void vcModals_DrawLoggedOut(vcState *pProgramState)
 }
 
 // Presents user with a message if the specified file exists, then returns false if user declines to overwrite the file
-bool vcModals_OverwriteExistingFile(vcState *pProgramState, const char *pFilename)
+bool vcModals_OverwriteExistingFile(vcState* pProgramState, const char* pFilename, const char* pFileExistingMsg)
```

Comment:
`*` go with the variable, I assume your VS is misconfigured?

Reply:
I guess so, FIxed